catalog-rest-service/src/main/resources/ui/src/pages/services/index.tsx

```diff
@@ -589,6 +659,18 @@ const ServicesPage = () => {
               </div>
             )}
 
+            {Boolean(!isNil(paging[serviceName].after)) && (
```

Comment:
isNil is already giving boolean value, so we do not need to cast it into `Boolean`